frame/bonded-finance/README.md

```diff
@@ -1,3 +1,68 @@
-# Overview
+# Bonded Finance
+
+A pallet providing means of submitting and maintaining bond offers.
+
+## Bond Offer Workflow
```

Comment:
should we have docs in rs and in md? I think right is one place. i do not care what.
also, I should that explanatory should go first - what is value of pallet and what is mechanic. example is good, but if you can do summary - ok. i do not think that details should go first. high level introductory should guide user reading later docs.

Reply:
As for your first suggestion, let me know what you think of these new changes.
As for the second, while I do see where you're coming from, I think there is value to having the readme and top-level cargo docs mirror each other in the case of pallets.